Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,611 @@
+material.
+
+The units scale is defined in such a way that level for Water is 0 and
+the level for Air is -1000.
+
+The attenuation of any other material is mapped linearly into this
+range.
+
+A table of Hounsfield units for common human tissues is avalable in the
+following link [ITK Hounsfield Units](https://itk.org/Wiki/ITK_Hounsfield_Units)
+
+### What are MRI T1 / T2 units?
+
+MRI images measure two types of relaxation times.
+
+The units are millisecons and the two relaxation concepts relate to how
+long it takes for the molecules in the tissue to realign themselves with
+a magentic field after being perturbed.
+
+Details on the MRI units are available on the [Relaxation (NMR)](https://en.wikipedia.org/wiki/Relaxation_%28NMR%29)
+Wikipedia page.
+
+### DICOM: Bits Allocated, Bits Stored and High Bit
+
+A question that is often asked on insight-users is: I am trying to
+read/write a DICOM image, but some of the DICOM fields have changed
+after I write it back on disk.
+
+Here is what you have on disk:
+
+```
+Bits Allocated = 16
+Bits Stored    = 12
+High Bit       = 11
+
+                  |<------------------ pixel ----------------->|
+    ______________ ______________ ______________ ______________
+   |XXXXXXXXXXXXXX|              |              |              |
+   |______________|______________|______________|______________|
+    15          12 11           8 7            4 3            0
+```
+
+Here is what you generally get after writing by GDCM:
+
+```
+Bits Allocated = 16
+Bits Stored    = 16
+High Bit       = 15
+
+   |<------------------------- pixel ------------------------->|
+    ______________ ______________ ______________ ______________
+   |              |              |              |              |
+   |______________|______________|______________|______________|
+    15          12 11           8 7            4 3            0
+```
+
+Since DICOM V3, you **cannot** store any overlay in the unused bits of
+the `Pixel Data` (`0x7fe0,0x0010`), thus it imply that the unused bits are
+**zeros**. In ITK, there is no such thing as 12-bit pixel type, thus
+when instanciating your reader you declare a 16-bit pixel type. This
+buffer is then passed back to GDCM which sees an array of 16bits values
+and then simply used the simpliest way to store this buffer back on
+disk: `Bits Stored = 16` and `High Bit = 15`. In case you have not made any
+change to your pixel buffer, implicitly the first 4 bits will be zero
+again. The only difference being that they are not as clearly declared
+as 12 bits.
+
+### I'm working with DICOM images, where can I find more information?
+
+See [ITK/DICOM](https://itk.org/Wiki/ITK/DICOM).
+
+## Data display
+
+### What imaging software is available for displaying medical image files?
+
+Please see the [third party applications](https://itk.org/Wiki/ITK/Third_Party_Applications) page for
+visualization applications that used ITK to perform image
+reading/writing.
```

Comment:
We should have a third-party applications page.

Reply:
PR #4313.

Missing
https://itk.org/Wiki/ITK/Related_Software
https://itk.org/Wiki/ITK/Developer_Software

from https://itk.org/Wiki/ITK/Other_Software